client/app/scripts/superdesk-dictionaries/dictionaries.js

```diff
@@ -143,7 +151,7 @@
                 $scope.origDictionary = result;
                 $scope.dictionary = _.create(result);
                 $scope.dictionary.content = _.create(result.content || {});
-                $scope.dictionary.is_active = $scope.dictionary.is_active !== false;
+                $scope.dictionary.is_active = $scope.dictionary.is_active !== 'false';
```

Comment:
Wouldn't it be better to convert the `is_active` flag to a boolean when data is fetched to the server? It's more error-prone if it stays as a string, e.g. somebody might do:

```js
if (!$scope.dictionary.is_active) {...}
```

...which would not execute because `!'false'` would evaluate to `false` instead of the expected `true`.